### DISCUSSION TITLE
Vyper Developer Advocate

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [Cyfrin/moccasin-full-course-cu#5] [Announcements]
Vyper is looking to hire a developer advocate!

People who finish the advanced curriculum, this is a great time to get your foot in the door!

https://x.com/vyperlang/status/1863970397038715141?s=46

**Comment by Scofield-Idehen** [comment]
I am more than available. I have completed the form.

**Comment by aditya98** [comment]
I am available, but is there any other position as well, because, I checked some web3 career pages and couldn't find anything with vyper.
